app/controllers/api/applications_controller.rb

```diff
@@ -52,11 +57,12 @@ def update
 
   def decide
     decision = params[:decision]
-    puts("DECISION")
-    puts(decision)
 
     begin
       application = Application.find(params[:id])
+      if decision == 2 and application.project.has_max_users
```

Comment:
don't refer to models by their ids correctly, these are not static

Reply:
do you want me to refer to it by their string name?

Reply:
PR #59 changes this to be an enum, so i'll change this on rebase once that PR gets merged in.